pkg/driver/controller.go

```diff
@@ -61,9 +61,10 @@ const isManagedByDriver = "true"
 
 // controllerService represents the controller service of CSI driver
 type controllerService struct {
-	cloud         cloud.Cloud
-	inFlight      *internal.InFlight
-	driverOptions *DriverOptions
+	cloud               cloud.Cloud
+	inFlight            *internal.InFlight
+	driverOptions       *DriverOptions
+	modifyVolumeManager *modifyVolumeManager
```

Comment:
I'm thinking renaming this to `coalescerManager` or something along those lines would be more descriptive, thoughts?